zerver/lib/url_preview/preview.py

```diff
@@ -10,7 +11,7 @@
 from zerver.lib.url_preview.parsers import OpenGraphParser, GenericParser
 
 
-CACHE_NAME = "database"
+CACHE_NAME = "database" if not settings.DEVELOPMENT else "in-memory"
```

Comment:
What's the story with this change?

Reply:
It's difficult to test changes to the code in development, because of the caching. Using an in-memory cache makes this easier, since each code change flushes the cache. (This was also listed as a potential option in the original issue #9451)

Reply:
I guess the commit message could use an improvement, if we agree this change is acceptable.